Sources/PubNubChatComponents/Message List/MessageListViewController.swift

```diff
@@ -82,6 +106,58 @@ open class ChatViewController: CollectionViewComponent {
     })
   }
   
+  open override func onLongPressGestureRecognized(gesture: UIGestureRecognizer) {
+    
+    if gesture.state == .began {
+      
+      // Finds the cell that was pressed
+      guard let affectedCell = collectionView.visibleCells.first(where: {
```

Comment:
when we should add new line after comment and when not? above before properties are added

Reply:
This is what we should discuss first I think. Or let any formatting tool to solve it